docs/.vale/thothtech/American.yml

```diff
@@ -0,0 +1,115 @@
+---
+# Error: thothtech.American
+#
+# Checks that British spelling is used instead of American spelling.
+#
+# For a list of all options, see https://errata-ai.gitbook.io/vale/getting-started/styles
+extends: substitution
+message: 'Use the British spelling "%s" instead of the American "%s".'
+link: TBA
+level: error
+ignorecase: true
+swap:
+  eon: aeon
+  airplane: aeroplane
+  aging: ageing
+  aluminum: aluminium
+  anemia: anaemia
+  anesthesia: anaesthesia
+  analyze: analyse
+  annex: annexe
+  apologize: apologise
+  authorize: authorise
+  authorized: authorised
+  authorization: authorisation
+  authorizing: authorising
+  behavior: behaviour
+  buses: busses
+  caliber: calibre
+  categorize: categorise
+  categorized: categorised
+  categorizes: categorises
+  categorizing: categorising
+  center: centre
+  check: cheque
+  civilization: civilisation
+  civilize: civilise
+  color: colour
+  cozy: cosy
+  cipher: cypher
```

Comment:
I think I would use cipher personally, but I am happy to be out voted on this

Reply:
Removed given rare and controversial usage.